k9mail/src/main/java/security/planck/sync/PlanckSyncRepository.kt

```diff
@@ -98,17 +98,13 @@ class PlanckSyncRepository @Inject constructor(
 
             SyncHandshakeSignal.SyncNotifySole -> {
                 isGrouped = false
-                if (syncState.allowSyncNewDevices) {
-                    startOrResetManualSyncTimer()
-                }
+                cancelIfCancelledByPartner()
             }
 
             SyncHandshakeSignal.SyncNotifyInGroup -> {
                 isGrouped = true
                 k9.markSyncEnabled(true)
-                if (syncState.allowSyncNewDevices) {
-                    startOrResetManualSyncTimer()
-                }
+                cancelIfCancelledByPartner()
```

Comment:
should  we also set the isGrouped to false in this case?

Reply:
You mean, in the case of key sync cancelled by partner?

Reply:
cancel should automatically ungroup on the engine, should it not?

Reply:
no, thats not the use case. Cancel means that it cancels the creation of the new group, but one of the devices could already be in the group and handshaking to add a new device to the same group.